src/data_projection_routines.f90

```diff
@@ -3608,5 +3656,55 @@ END SUBROUTINE DATA_PROJECTION_RESULT_XI_SET
   !================================================================================================================================
   !
 
+  !>Gets the projection vector for a data point identified by a given global number.
+  SUBROUTINE DATA_PROJECTION_RESULT_PROJECTION_VECTOR_GET(DATA_PROJECTION,DATA_POINT_USER_NUMBER,PROJECTION_VECTOR,ERR,ERROR,*)
```

Comment:
You should use the new CamelCase code style for new routines rather than the older uppercase style.